codice/ddf
# [2.19.x] G-8880 Fixed Antarctica Geometry

#### What does this PR do?
Fixes the geometry for Antarctica

#### Who is reviewing it? 
@andrewzimmer 
@mojogitoverhere 
@Lambeaux 

#### Ask 2 committers to review/merge the PR and tag them here.
@andrewzimmer
@lambeaux
@mojogitoverhere

#### How should this be tested?
This should be tested on a downstream project that utilizes the gazetteer. Please contact me before testing to ensure you grant the downstream project permissions to access the gazetteer data.
Build/Install development profile
Navigate to Intrigue
In the gazetteer search text box, enter "Antarctica"
Verify you are suggested the feature for Antarctica
Click the Antarctica Feature and verify the map pans to Antarctica

#### Any background context you want to provide?
The geometry being used for Antarctica was not working in the gazetteer due to limitations in geo libraries that DDF uses which are not able to handle polygons that wrap around the poles. This modifies the geometry for Antarctica so that Antarctica now works correctly when selected in the gazetteer.

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
